.serena/memories/pr-changes-requested-semantics.md

```diff
@@ -0,0 +1,68 @@
+# CHANGES_REQUESTED Semantics for Bot-Authored PRs
```

Comment:
The protocol is sufficiently complicated that it needs to be documented outside of a memory and the memory updated to refer to the documentation. Documentation must contain a mermaid diagram for state descriptions.